docs/src/pages/components/data-grid/filtering/filtering.md

```diff
@@ -4,162 +4,240 @@ title: Data Grid - Filtering
 
 # Data Grid - Filtering
 
-<p class="description">Filtering helps to view a subset of the records based on a criteria.</p>
+<p class="description">Easily filter your rows based on one or several criteria.</p>
 
-## Basic filter
+The filters can be modified via through grid interface in several ways:
```

Comment:
small minor fix to replace "via through".
The filters can be modified through the grid interface in several ways:

Reply:
Fixed :+1: